cmd/conformance/main.go

```diff
@@ -93,33 +101,135 @@ func getTrustedRoot(staging bool) root.TrustedMaterial {
 	return tr
 }
 
+func parseArgs() {
+	for i := 2; i < len(os.Args); {
```

Comment:
Could we use [flags](https://pkg.go.dev/flag) rather than parsing the arguments?

Reply:
SG